executorCommon/src/main/resources/application.conf

```diff
@@ -1,5 +1,5 @@
 dxExecutorCommon {
-    version = "2.6.1-SNAPSHOT"
+    version = "2.7.0"
```

Comment:
Please update working version to 2.7.1-SNAPSHOT for `develop`. Maybe create a new branch starting from `release-2.7.0` and make the PR from that. @jdidion

Reply:
done

Reply:
I meant make the other PR https://github.com/dnanexus/dxCompiler/pull/227 (which looks like it has this branch's contents + the version update) into `develop` and replace this PR with that one. @jdidion

Reply:
let's do them as two separate PRs.

Reply:
ok.